source/octod/core.d

```diff
@@ -175,12 +175,14 @@ struct HTTPConnection
 
         Params:
             url = GitHub API method URL (relative)
+            accept = optional argument with custom `Accept` header to
+                specify for this one request
 
         Returns:
             Json body of the response. If response is multi-page, all pages
             are collected and concatenated into one returned json object.
      **/
-    Json get ( string url )
+    Json get ( string url, string accept = "" )
```

Comment:
Silly question, but isn't `null` a better representation for the lack of a value?
Also, I don't see where this new `accept` is being used :thinking: